cli/internal/packager/create.go

```diff
@@ -75,7 +75,7 @@ func addLocalAssets(tempPath componentPaths, assets config.ZarfComponent) {
 	if len(assets.Charts) > 0 {
 		logrus.Info("Loading static helm charts")
 		_ = utils.CreateDirectory(tempPath.charts, 0700)
-		re := regexp.MustCompile("\\.git$")
+		re := regexp.MustCompile(`\.git$`)
```

Comment:
Was this breaking before? I almost expect to see some other changes to how `.git` files are being written when looking at this change.

Reply:
`pre-commit` picked it up as a lint error. It worked fine before, and should still work fine, just without throwing a lint error.